cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/LoginCommandOptions.java

```diff
@@ -19,15 +19,15 @@
         description = "Your password of your cf account.")
     String password;
 
-    @Option(names = { "-a", "--api" }, required = false, scope = ScopeType.INHERIT,
+    @Option(names = { "-a", "--api" }, required = true, scope = ScopeType.INHERIT,
```

Comment:
I wouldn't make these changes now, given we want to reimplement/port the `target` YAML key support. Maybe put `TODO`s there if you really want to change these. However, I think that handling missing options is implemented in our code already, we don't really need PicoCLI to handle those even until `target` is brought back.